1-js/11-async/08-async-await/article.md

```diff
@@ -224,81 +226,82 @@ async function f() {
     let response = await fetch('http://no-such-url');
   } catch(err) {
 *!*
-    alert(err); // TypeError: failed to fetch
+    alert(err); // TypeError: gagal untuk mengambil
```

Comment:
Errornya jangan diterjemahin gan, kasih catatan tambahan aja TypeError: failed to fetch (gagal mengambil resource). Soalnya itu keluaran dari Javascriptnya... hehe

Reply:
siap mas. astaga lupa. saya update dulu

Reply:
Mantap, santai